examples/mysql-wordpress-pd/wordpress-deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: wordpress
+  labels:
+    app: wordpress
```

Comment:
I have `app: wordpress` on all the items. On the pods I have `tier: frontend`, but not the deployment, claim, and service. Should I have `tier: frontend` on all those?